pkg/types/types.go

```diff
@@ -261,10 +265,28 @@ func MergeMetrics(metrics [][]Metric, replicaMismatchConfig cfg.RenderReplicaMis
 			zap.Int("replica_points_total", metricsStat.DataPointCount),
 		)
 	}
+	if len(diffResolutionMetrics) > 0 {
```

Comment:
Do we know how frequently this happens in real life? If this happens a lot, this may significantly impact the volume of logs. Have we tested for this?

Have we considered adding a metric instead of a log record? Do we need the details in the log record, or would a metric suffice?

Reply:
👍 for live test, impact can be significant, agreed